### ISSUE TITLE
openidConnectClient-1.0 feature is not compatible with mpHealth-1.0

### ISSUE THREAD

**Issue by iamahern** [OpenLiberty/open-liberty#3983]
The dependencies of openidConnectClient-1.0 and mpHealth-1.0 clash, preventing both features from being used in a single deployment.

Obvious uses cases where this comes up is if you are deploying Liberty into a Kubernetes / Container environment and you are integrating with IBM AppID [ahem].

Issue was found using Liberty 18.0.0.1

Trace is is a follows:
[ERROR   ] CWWKF0033E: The singleton features com.ibm.websphere.appserver.javax.el-2.2 and com.ibm.websphere.appserver.javax.el-3.0 cannot be loaded at the same time.  The configured features openidConnectClient-1.0 and mpHealth-1.0 include one or more features that cause the conflict. Your configuration is not supported; update server.xml to remove incompatible features.

-----
I see that issue #3690 is refactor this area, so that will likely resolve the issue. 

A fix [update the feature dependencies] would be desired ASAP depending on the timeline of #3690.

**Comment by markecock** [comment]
+1 on this..an expedient fix for this would be much appreciated.

**Comment by aguibert** [comment]
Using the latest WAS Liberty beta (2018.6.0.0) this no longer appears to be an issue.  Namely, I was able to start up the following feature set without any errors:

```xml
    <featureManager>
        <feature>openidConnectClient-1.0</feature>
        <feature>mpHealth-1.0</feature>
    </featureManager>
```

**Comment by iamahern** [comment]
Looks fine for me as well.

**Comment by aguibert** [comment]
Thanks for confirming @iamahern, marking this issue fixed in `release-18.0.0.2` and closing
